### PR TITLE
Added remaining repos for syncing

### PR DESCRIPTION
**Pull request by kenodegard**
Depends on #25

### CODE REVIEW

**Comment by kenodegard** [comment]
@rchord @jezdez @LtDan33 @beeankha @awwad @chenghlee 

Removed the archived [conda/conda_build_single_folder_test](https://github.com/conda/conda_build_single_folder_test) repo from syncing.

What do we think is the future of the [conda/conda_build_test_recipe](https://github.com/conda/conda_build_test_recipe)? This fits the same pattern as the archived repo however it has had more recent activity (dependabot was enabled). Should we go ahead and lock this repo down as well (disable issues/actions/etc.) and possibly archive it?

We should specify in the README for both of these that any necessary changes should be reported to [conda/conda](https://github.com/conda/conda) and we'd unarchive/modify the repos as necessary from there?

Outside of these two test repos, are any of the other repos in the conda org that we should potentially treat differently from the others wrt triaging/labels/automations?

**Comment by rchord** [comment]
I haven't heard about wanting to archive the conda/conda_build_test_recipe repo, so I'm not sure what the ramifications are of that since conda-build relies on that. It sounds like it will still be available but not editable?

**Comment by kenodegard** [comment]
@rchord correct, when archived it becomes read-only, the other repo [conda/conda_build_single_folder_test](https://github.com/conda/conda_build_single_folder_test) is also used in conda-build's test suite

File: .github/sync.yml [hunk 1]
```diff
@@ -1,7 +1,25 @@
 group:
   - repos: |
+      conda/actions
+      conda/ceps
```
Comment (kenodegard) [inline .github/sync.yml:4]: Do not include CEPS

File: .github/sync.yml [hunk 2]
```diff
@@ -1,7 +1,25 @@
...
+      conda/clabot-config
```
Comment (kenodegard) [inline .github/sync.yml:5]: Do not include clabot-config

File: .github/sync.yml [hunk 3]
```diff
@@ -1,7 +1,25 @@
...
       conda/conda
+      conda/conda-benchmarks
+      conda/conda-bots
```
Comment (kenodegard) [inline .github/sync.yml:8]: Do not include conda-bots

File: .github/sync.yml [hunk 4]
```diff
@@ -1,7 +1,25 @@
...
       conda/conda-build
+      conda/conda-content-trust
+      conda/conda-docs
+      conda/conda-pack
+      conda/conda-package-handling
+      conda/conda-prefix-replacement
+      conda/conda-verify
+      conda/conda_build_test_recipe
```
Comment (kenodegard) [inline .github/sync.yml:16]: Don't include conda_build_test_recipe since we will transition this into an archived repo

File: .github/sync.yml [hunk 5]
```diff
@@ -1,7 +1,25 @@
...
+      conda/constructor
+      conda/cookiecutter-conda-python
+      conda/issue-tracker
```
Comment (kenodegard) [inline .github/sync.yml:19]: Drop issue-tracker

**Review by jezdez — APPROVED** [review]
Good one, we discussed this in person.

**Pull request merged**